client/browser/src/shared/code-hosts/github/util.tsx

```diff
@@ -207,9 +210,10 @@ function getDiffResolvedRevisionFromPageSource(
  * TODO ideally, this should only scrape the code view itself.
  */
 export function getFilePath(): string {
-    const permalink = document.querySelector<HTMLAnchorElement>('a.js-permalink-shortcut')
+    const selector = getPermalinkSelector()
+    const permalink = document.querySelector<HTMLAnchorElement>(selector!)
```

Comment:
```suggestion
    const permalink = document.querySelector<HTMLAnchorElement>(selector)
```
Handle not existing selector appropriately